alembic/versions/1a9f26a69a46_initial_database_schema.py

```
@@ -0,0 +1,162 @@
+"""Initial database schema
+
+Revision ID: 1a9f26a69a46
+Revises: 
+Create Date: 2025-07-10 16:53:21.384073
+
+"""
+from typing import Sequence, Union
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)